utils/AlertGenerator.py

```diff
@@ -4,8 +4,15 @@
 from mq.AlertEvent import emit_alert_event
 
 
-quarantined_alerts = ["LAF-002", "LAF-006", "LAF-007", "LAF-009", "LAF-100", "LAF-101"]
-
+alert_blocked_by = {
+    "LAF-002" : ["LAF-002"],
+    "LAF-006" : ["LAF-006"],
+    "LAF-007" : ["LAF-007"], 
+    "LAF-009" : ["LAF-009"],
+    "LAF-100" : ["LAF-100"],
+    "LAF-101" : ["LAF-101"],
+    "LAF-501" : ["LAF-501", "LAF-404"]
+}
```

Comment:
Is not necessary to have an entry with key "LAF-404"?

Reply:
Yes! LAF-404 blocks itself. Thanks